PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/component/input/ComboBoxResourceGroup.java

```diff
@@ -45,8 +45,12 @@ protected String getItemText(final Object item) {
         return ((ResourceGroup) item).name();
     }
 
-    public void refreshWith(Subscription subscription) {
+    public void setSubscription(Subscription subscription) {
         this.subscription = subscription;
```

Comment:
two many duplicate setSubscription method

Reply:
yes, but java doesn't support multiple class inheritance.
do you have any idea to solve this problem?